### PR TITLE
feat(nix): introduce `juno-subql` and cosmos network

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**

### CODE REVIEW

**Review by dzmitry-lahoda — APPROVED** [review]

File: .nix/services/subsquid-indexer-status-service.nix [hunk 1]
```diff
@@ -5,7 +5,7 @@
     network_mode = "host";
     restart = "unless-stopped";
     environment = {
-      REDIS_URL = "redis://localhost:6379/0";
+      REDIS_URL = "redis://1270.0.1:6379/0";
```
Comment (aeryz) [inline .nix/services/subsquid-indexer-status-service.nix:8]: 127.0.0.1

File: .nix/arion-xcvm.nix [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        config.services = {
+
+          # ============ COMMON ===============
+          db = import ./services/postgres.nix {
+            database = default-db;
+            version = "14";
+            init-scripts = pkgs.writeTextFile {
+              name = "init";
+              text = ''
+                CREATE DATABASE juno WITH OWNER xcvm;
```
Comment (aeryz) [inline .nix/arion-xcvm.nix:30]: Don't know the nix language exactly but can we change these owner names to "database.user". Eg in rust:
```rust
format!("... OWNER {}", database.user);
```

File: .nix/services/subsquid-indexer-status-service.nix (hunk 1 above)
Comment (hussein-aitlahcen) [reply .nix/services/subsquid-indexer-status-service.nix]: done

File: .nix/arion-xcvm.nix (hunk 2 above)
Comment (hussein-aitlahcen) [reply .nix/arion-xcvm.nix]: done

**Review by aeryz — APPROVED** [review]

**Review by dzmitry-lahoda — APPROVED** [review]

**Pull request merged**